pkg/util/mount/mount.go

```diff
@@ -50,7 +45,12 @@ type MountPoint struct {
 	Pass   int
 }
 
-// Examines /proc/mounts to find all other references to the device referenced
+// New returns a mount.Interface for the current system.
+func New() Interface {
```

Comment:
No real value, it was there before.  (I moved the order though, to make type declaration before method declaration).

I think we can leave it there.  The New() method hides interface implementation from user, so client code doesn't need to know what type implements the interface.  Everything is defined in mount.go.